docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Some comments on this page: 
https://github.com/SimpleDataLabsInc/prophecy-docs/pull/307/files

Reply:
I incorporated most of your changes. Thank you